ios/ElectrodeApiImpl/APIImpls/Navigation/ENNavigationDelegate.swift

```diff
@@ -37,7 +37,7 @@ import UIKit
                 miniappVC.navigateWithRoute = vc.navigateWithRoute
                 miniappVC.globalProperties = vc.globalProperties ?? nil
                 navigationVC.navigationBar.isTranslucent = false
-                navigationVC.pushViewController(miniappVC, animated: false)
+                navigationVC.pushViewControllerWithoutBackButtonTitle(miniappVC, animated: false)
```

Comment:
Will this be the default behavior ?

Reply:
yes it is.

Reply:
I don't think we should make this a default behavior. This should be a custom option for apps that wants to have a root component with a back option. If they decide to run the app independently the first page should not have a back/cancel option.

Reply:
This PR is not about the first page back button. It's about removing the title in back button. I've updated before and after images in the description.

Reply:
thanks for the clarification.